### DISCUSSION TITLE
Obsidian Clipper Chrome Extension Stopped Working

### DISCUSSION THREAD

**Discussion by mark0mark0mark** [jgchristopher/obsidian-clipper#84] [General]
I had been using the Obsidian Clipper extension for many months without issue.  I am not using Daily Notes, but Topic Notes.
Recently, the Chrome extension stopped working.  When I click on it, nothing is added to Obsidian.

This is the page I was attempting to capture:
https://medium.com/obsidian-observer/organising-your-notes-in-obsidian-a-complete-guide-bf939d7e47e8

After I click on the Clipper icon in the extensions toolbar in Chrome:
In the Chrome Console, I see what is in the attached file.
In the Obsidian, I see nothing in the Obsidian debug Console.

Appreciate any help in figuring out what is going wrong.

Thanks,

Mark
[new_medium.com-1735572643220.log](https://github.com/user-attachments/files/18275988/new_medium.com-1735572643220.log)

**Comment by jgchristopher** [comment]
Hey Mark,

I just tried to reproduce this in both my daily note and a new topic note and I was able to clip data successfully on that page you linked. 

Was there a specific section of the page that you were trying to clip? Or all of it?

I am also using the beta version of clipper, so not sure if that is playing a role as well. 

Thanks,
John

**Comment by mark0mark0mark** [comment]
Thanks for the reply, John.

I was only trying to clip a couple of paragraphs.  But it doesn't work on
any web page I try.  Even if I don't highlight anything on the page.

Are you saying there's a beta version of the Chrome extension or of the
Obsidian plugin?  And where do I get that?

Thanks,

Mark
